pubsub/noxfile.py

```diff
@@ -78,7 +78,7 @@ def default(session):
         "--cov-append",
         "--cov-config=.coveragerc",
         "--cov-report=",
-        "--cov-fail-under=97",
+        "--cov-fail-under=0",
```

Comment:
Sanity check - we are proceeding with this? I saw [the issue](https://github.com/googleapis/google-cloud-python/issues/8174) mentioning the problem with any threshold below 100 %, but just want to make sure that this is indeed an intentional change.

Reply:
:+1: We merged the template change yesterday to synthtool.